packages/ui/src/validators/components/ValidatorsList.tsx

```diff
@@ -19,8 +19,10 @@ export const ValidatorsList = ({ validators }: ValidatorsListProps) => {
         <ListHeader>Validator</ListHeader>
         <ListHeader>Verification</ListHeader>
         <ListHeader>State</ListHeader>
-        <ListHeader>Total Reward</ListHeader>
-        <ListHeader>APR</ListHeader>
+        <ListHeader>Own Stake</ListHeader>
+        <ListHeader>Total Stake</ListHeader>
+        <ListHeader>Expected Nom APR</ListHeader>
+        <ListHeader>Comission</ListHeader>
```

Comment:
The table should be sort-able by "Expected Nom APR" and "Commission"

![image](https://github.com/Joystream/pioneer/assets/6571453/c4583d9d-7415-4ec2-8867-bac6b250f31d)